apps/api/src/main.ts

```diff
@@ -42,11 +46,32 @@ class CustomLogger implements LoggerService {
   }
 }
 
-async function bootstrap() {
+async function initializeSentry() {
+  if (!process.env.SENTRY_DSN) {
```

Comment:
> The Sentry initialization checks for SENTRY_DSN but does not validate other Sentry-related environment variables like SENTRY_ORG, SENTRY_PROJECT, and SENTRY_AUTH_TOKEN, which are crucial for source map uploads and might cause runtime errors if missing or incorrect.

I think this is the last change that needs to be incorporated . after this, we are good to go!

Reply:
Ok,  noted.